features/communities/events/EventForm.tsx

```diff
@@ -138,6 +138,7 @@ export default function EventForm({
             event.offlineInformation.lng,
             event.offlineInformation.lat
           ),
+          bbox: [0, 0, 0, 0] as [number, number, number, number]
```

Comment:
```suggestion
          bbox: [0, 0, 0, 0] as const
```
Slightly less code to write!

Reply:
that gave problems in other part 🙃